core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -277,6 +279,20 @@ public static boolean throwUnless(boolean condition, String message) {
     return condition;
   }
 
+  public static String uuidToString(UUID uuid) {
```

Comment:
Shall we add some tests in `SqlFunctionsTest`?

Reply:
These functions are so small that I didn't think there is much to test about them. They essentially delegate all the work to the library functions.